internal/cmd/billing/command_cost_test.go

```diff
@@ -0,0 +1,102 @@
+package billing
+
+import (
+	"context"
+	billingv1 "github.com/confluentinc/ccloud-sdk-go-v2/billing/v1"
+	billingMock "github.com/confluentinc/ccloud-sdk-go-v2/billing/v1/mock"
+	"github.com/confluentinc/cli/internal/pkg/ccloudv2"
+	v1 "github.com/confluentinc/cli/internal/pkg/config/v1"
+	climock "github.com/confluentinc/cli/mock"
+	"github.com/spf13/cobra"
+	"github.com/stretchr/testify/require"
+	"github.com/stretchr/testify/suite"
+	"net/http"
+	"testing"
+)
+
+type CostTestSuite struct {
+	suite.Suite
+	conf            *v1.Config
+	billingCostMock *billingMock.CostsBillingV1Api
+}
+
+var costList = billingv1.BillingV1CostList{
+	Data: []billingv1.BillingV1Cost{
+		{
+			StartDate:   billingv1.PtrString("2023-01-01 00:00:00"),
+			EndDate:     billingv1.PtrString("2023-01-02 00:00:00"),
+			Granularity: billingv1.PtrString("DAILY"),
+			LineType:    billingv1.PtrString("LINE_TYPE_1"),
+			Product:     billingv1.PtrString("KAFKA"),
+			Resource: &billingv1.BillingV1Resource{
+				Id:          billingv1.PtrString("lkc-123"),
+				DisplayName: billingv1.PtrString("kafka_1"),
+			},
+			NetworkAccessType: billingv1.PtrString("INTERNET"),
+			Price:             billingv1.PtrFloat64(0.123),
+			Unit:              billingv1.PtrString("GB"),
+			Amount:            billingv1.PtrFloat64(50.0),
+			OriginalAmount:    billingv1.PtrFloat64(50.0),
+		},
+	},
+}
+
+func (suite *CostTestSuite) SetupTest() {
```

Comment:
Ah, sorry to have you delete your code, but we're switching to a new (easier to write) form of integration tests, which you can find in the `test/` directory.

Reply:
thanks @brianstrauch  i should've read the contribution guide :)

Reply:
No worries! Hopefully the other tests will be significantly easier to write.